libpod/container_api.go

```diff
@@ -158,7 +158,7 @@ func (c *Container) Update(resources *spec.LinuxResources, restartPolicy *string
 	}
 
```

Comment:
Maybe should error if the container is running, as the environment variables won't be updated until the container restarts? Or warn instead? Would like more opinions on this.

Reply:
I don't think it should error or warn. But it should be clearly documented that env updates only effect the main container process after the next start.
I am not quite sure on our logic but it seems podman exec copies the env from the main container so I assume it would get picked up by exec right away?

Reply:
I agree with @Luap99 here, but I see that people might be confused by updating a env variable and thinking it will change inside of the already running process in a container.

Maybe it could print some info in case the container is running? Would that be acceptable, or should I just improve the documentation?

Reply:
I've so far just improved the documentation. Tell me if you think some special note in the podman update output is needed.